tests/type_inference/test_functions.py

```diff
@@ -0,0 +1,57 @@
+import astroid
+import nose
+from hypothesis import assume, given
+import tests.custom_hypothesis_support as cs
+import hypothesis.strategies as hs
+from typing import Callable
+from python_ta.transforms.type_inference_visitor import register_type_constraints_setter,\
+    environment_transformer, TYPE_CONSTRAINTS
+from keyword import iskeyword
+
+
+def _parse_to_function(function_name, args_list, return_value):
+    """Helper to parse given data into function definition."""
+    return "def " + function_name + "(" + ", ".join(args_list) + "):\n\t" + "return " + repr(return_value)
```

Comment:
Since we're pretty locked into Python 3.6 now, you can start using f-strings (https://www.python.org/dev/peps/pep-0498/). This will make these helper functions a lot easier to read. Pretty sure you can do multiline f-strings too, so you don't need to hard-code a newline or indentation character.